gateway_config.yml

```diff
@@ -2,7 +2,7 @@ environment:
   validate_hmac: "1"
 # URLs
   gateway_url:  http://gateway.openfaas:8080/
-  gateway_public_url: https://cloud.o6s.io/
+  gateway_public_url: https://system.o6s.io/
```

Comment:
There is no need to change this, because this references to the public URL in which your OpenFaaS instance is running

Reply:
Thanks @martindekov. I edited the fix. Can you please review?